src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentMetadataGeneral.js

```diff
@@ -70,9 +70,7 @@ export class DocumentMetadataGeneral extends Component {
           <List bulleted>
             {document.metadata.alternative_titles.map((entry) => (
               <List.Item key={entry.value}>
-                <List.Content>
-                  {entry.value + '(' + entry.type + ')'}
```

Comment:
We need to show title type if present, please add it back. especially if it is librarians view, they need all the info displayed